src/dom.ts

```diff
@@ -69,6 +76,13 @@ export const registerDom = <ElementType extends TramOneHTMLElement | TramOneSVGE
 			// cleanup effects will be populated when new effects are processed
 			tagResult[TRAM_TAG_CLEANUP_EFFECTS] = [];
 
+			// save properties for development debugging
+			if (process.env.NODE_ENV === 'development') {
+				tagResult[TRAM_TAG_NAME] = tagName;
+				tagResult[TRAM_TAG_PROPS] = props;
+				tagResult[TRAM_TAG_CHILDREN] = children;
+			}
```

Comment:
these properties are not otherwise on the tag, so we need to add them (but only for development).

`tagName` here is based on what the parent component used to add this to the registry.